.gitignore

```diff
@@ -32,4 +32,4 @@ Cargo.lock
 
 # Ignore DataStore and Database files
 *.mdb
-.data/
+data
```

Comment:
Doesn't this ignore all folders called bin

Reply:
Woops I meant to just take the dot off the front. Will fix